gha-cache/src/api.rs

```diff
@@ -624,10 +624,16 @@ impl AtomicCircuitBreaker for AtomicBool {
     fn check_err(&self, e: &Error) {
         if let Error::ApiError {
             status: reqwest::StatusCode::TOO_MANY_REQUESTS,
-            info: ref _info,
+            ..
         } = e
         {
             tracing::info!("Disabling GitHub Actions Cache due to 429: Too Many Requests");
+            let msg = "\
+                Magic Nix Cache has exceeded GitHub Actions Cache's rate limit. \
+                Save more time and seamlessly share the builds across your team with FlakeHub Cache. \
+                See: https://flakehub.com/pricing\
+            ";
+            println!("::notice::{msg}");
```

Comment:
```suggestion
            let title = "GitHub Actions' cache is too small for your Nix project.";
            let msg = "\
                Turn Magic Nix Cache into a cache for your whole team. \
                Move beyond GitHub's limits, save time, and share builds outside CI with FlakeHub Cache. \
                See: https://dtr.mn/github-cache-limits\
            ";
            println!("::notice title={title}::{msg}");
```

![CleanShot 2025-01-15 at 17 46 14@2x](https://github.com/user-attachments/assets/4e280729-db01-458f-b3c4-2ce5da07670e)

![image](https://github.com/user-attachments/assets/09fbb091-d6c8-4682-8b41-593e0989c654)